templates/books/book-detail.html

```diff
@@ -50,11 +50,6 @@ <h1 class="h2">{{ book.title }}</h1>
                 <div class="mb-2">{{ book.title_ru }}</div>
             {% endif %}
 
-            <!--Book Duration-->
-            {% if book.duration_sec %}
-                <p>{{ book | duration }}</p>
-            {% endif %}
```

Comment:
Depending on whether book has a single or multiple narrations we use slightlhy different layouts. For example to render narrator:

* with single narration we render narrator next to author (book level): https://audiobooks.by/books/rukh

* with multiple narrations we render narrators at the narration level: https://audiobooks.by/books/liudzi-na-balotse

We should do the same with duration. The rationale is that displaying duration at book level visually better and majority of the books 95+% have only single narration. So we should optimize for that case.

Reply:
I changed a bit template, so please check again. Is that what you mean?

Reply:
Yes, but left comment below.